rust/integration-tests/src/diff_test_helper/event_processor.rs

```diff
@@ -0,0 +1,30 @@
+use crate::models::queryable_models::Event;
+use anyhow::Result;
+use diesel::{
+    pg::PgConnection,
+    query_dsl::methods::{FilterDsl, ThenOrderDsl},
+    ExpressionMethods, RunQueryDsl,
+};
+use processor::schema::events::dsl::*;
+use serde_json::Value;
+use std::collections::HashMap;
+
+#[allow(dead_code)]
```

Comment:
Should we delete all the dead code? If this is only used in test, we can add `#[cfg(test)]` so that it only compiles for tests.

Reply:
actually they are being used, but i am not sure why the lint complains them not being used..

Reply:
nvm, we can add #[cfg(test)]

Reply:
Try adding `#[cfg(test)]` so compiler knows this should only be compiled with tests